ZelBack/src/services/syncthingService.js

```diff
@@ -2108,7 +2109,7 @@ async function installSyncthing() { // can throw
     const exec = `cd ${nodedpath} && bash installSyncthing.sh`;
     await cmdAsync(exec);
   }
-  syncthingInstalled = true;
+  syncthingExecutable = true;
   log.info('Syncthing installed');
```

Comment:
Doesnt look like syncthing old version enforcement is working... plenty of nodes on the old versions
{'v1.27.6': 11192, 'v1.27.7-rc.1': 2, 'v1.27.5': 54, 'v1.27.4': 51, 'v1.25.0': 8, 'v1.27.3': 5, 'v1.23.1': 2, 'v1.27.2': 20, 'v1.27.1': 8, 'v1.27.0': 2, 'v1.23.4': 4, 'v1.24.0': 2, 'v1.23.6': 1, 'v1.23.2': 1, 'v1.22.1': 1, 'v1.26.1': 5, 'v1.22.2': 1, 'v1.23.7': 1}

Reply:
We're not enforcing it. (yet)